docs/topics/secured-mission-intro.adoc

```diff
@@ -1,15 +1,65 @@
 = Project Structure
-The project is split into two Apache Maven modules:
 
-app:: 
-The `app` module exposes the REST Service using {runtime-name}.
+The SSO booster project contains:
+* the sources for the Greeting service, which is the one which we are going to to secure
+* a template file (`service.sso.yaml`) to stand up the SSO server 
+* the Keycloak adapter configuration to secure the service
+
+= Standing up {RHSSO}
+The `service.sso.yaml` file contains all OpenShift configuration items to stand up a pre-configured {RHSSO} server. The the SSO server configuration has been simplified for the sake of this exercise and does provide an out-of-the-box configuration, with pre-configured users and security settings. 
+
+IMPORTANT: It is not recommended to use this SSO configuration in production. Specifically, the simplifications made to the booster security configuration impact the ability to use it in a production environment.
+
+.SSO Booster Simplifications
+[width="100%",options="header"]
+|====================================================================
+|Change |Reason |Recommendation
+
+a|
+The default configuration includes both public and **private keys in the yaml configuration files**.
+
+a|
+We did this because the end user can deploy Red Hat SSO module and have it in a usable state without needing to know the internals or how to configure Red Hat SSO.
+
+a|
+In production, it is recommend to not store any private key under source control and have it added by the server administrator.
+
+
+a|
+The configured **clients accept any callback url**.
+
+a|
+To avoid having a custom configuration per runtime we avoid the verification of the callback as mandated by the OAuth2 specification.
+
+a|
+A application specific callback URL should be provided with a valid domain name.
+
+a|
+**Clients do not require SSL/TLS**.
+
+a|
+The boosters are simplified by not requiring certificates generated for each runtime.
+
+a|
+In production a secure application should use HTTPS over plain HTTP.
+
+a|
+**The token timeout has been increased to 10 minutes from the default of 1 minute.**
+
+a|
+Provides a better user experience when working with the command line examples
+
+a|
+From a security perspective, the window an attacker would have to guess the access token is extended. It is recommended to keep this window short as the rotation creates entropy to guess the current token.
```

Comment:
"rotation creates entropy to guess the current token" is a mishmash of words :-) I'd reword to "it is recommended to keep this window short, which makes it much harder for a potential attacker to guess the current token".